public/instances.json

```diff
@@ -0,0 +1,72 @@
+{
+  "instances": [
+    {
+      "name": "Contrib",
+      "image": "/images/instances/contrib.png",
```

Comment:
> Since we already define dynamically the `id` (and thus all links) of the instance from its `kebabCase(name)`, I wouldn't be shocked if we also assumed the `image` is always at `/images/instances/${id}.png` 😉

Reply:
https://github.com/ambanum/OpenTermsArchive.org/pull/161/commits/e19159902f322d612c392ae1eea25f4f0c4dfa08